source/application/IDEAS/IDEASAppController.cpp

```diff
@@ -156,7 +160,7 @@ void IDEASAppController::setupUserInterface()
 
 	mw_->insertHeading("XRF Detectors", 1);
 
-	ideasKETEKDetailedDetectorViewWithSave_ = new IDEASXRFDetailedDetectorViewWithSave(IDEASBeamline::ideas()->ketek());
+	IDEASKETEKDetailedDetectorView *ideasKETEKDetailedDetectorViewWithSave_ = new IDEASKETEKDetailedDetectorView(IDEASBeamline::ideas()->ketek());
```

Comment:
you should remove `IDEASKETEKDetailedDetectorView *`. Otherwise, it will make `ideasKETEKDetailedDetectorViewWithSave_` a local variable instead a class property.